### PR TITLE
Notification component added for popup notifications

### PR DESCRIPTION
**Pull request by mavisakalyan**
## Describe your changes

- Added a notification component which appears when you hide or show the UI tools using CTR+H or the hide button in the actions menu.
NOTICE: This component is now dynamic and can be used from ANYWHERE. Just import "setPopupNotification" hook from the AppContext and whenever you set a message in any use case it will appear and disappear after 5 seconds. 

## What are the steps for a QA tester to test this pull request?

- Click the hide button in the actions menu and see the notification appear and disappear after 5 seconds.
- Same when pressing the CTR+H keys on the keyboard 

## Issue ticket number and link

Issue: 3394
https://github.com/webaverse/app/issues/3394

## Screenshots and/or video

https://user-images.githubusercontent.com/55106546/183292950-fb637d3d-77dc-4af5-b877-249fe2bf5a27.mov

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I am not adding any irrelevant code or assets
- [x] I am only including the changes needed to implement the change
- [x] I have playtested and intentionally tried to find error cases but couldn't

### CODE REVIEW

**Comment by lalalune** [comment]
Text should be:
"Press CTRL + H to unhide UI"
Should fade out and should for 3 seconds with a .5 second fade

**Comment by mavisakalyan** [comment]
@lalalune Made the changes, pleas review and proceed.

File: src/components/general/action-menu/ActionMenu.jsx [hunk 1]
```diff
@@ -61,6 +61,9 @@ export const ActionMenu = ({ setUIMode, className }) => {
     const handleModeBtnClick = () => {
 
         setUIMode( uiMode === 'normal' ? 'none' : 'normal' );
+        if(uiMode === 'normal') {
```
Comment (avaer) [inline src/components/general/action-menu/ActionMenu.jsx:64]: Spacing issue on `if`. Please follow the lint rules.

File: src/PopupNotification.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    if(popupNotification) {
+      const timeoutFade = setTimeout(() => {
+        ref.current.style.opacity = 0;
+      }, 3000)
+      const timeout = setTimeout(() => {
+        ref.current.style.opacity = 0;
+        setPopupNotification(null);
+      }, 3500)
+      return () => {
+        clearTimeout(timeout,timeoutFade);
```
Comment (avaer) [inline src/PopupNotification.jsx:27]: `clearTimeout` takes 1 argument.

File: src/PopupNotification.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import styles from "./PopupNotification.module.css";
+import { AppContext } from "./components/app";
+
+const PopupNotification = function ({
+  popupNotification
+}) {
+  const ref = React.useRef();
+  const { setPopupNotification } = React.useContext( AppContext );
+
+  React.useEffect(() => {
```
Comment (avaer) [inline src/PopupNotification.jsx:11]: Poor spacing.

File: src/PopupNotification.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  if(popupNotification) {
+    ref.current.style.opacity = 1;
+  }
+}, [popupNotification])
+
+  React.useEffect(() => {
+    if(popupNotification) {
+      const timeoutFade = setTimeout(() => {
+        ref.current.style.opacity = 0;
+      }, 3000)
```
Comment (avaer) [inline src/PopupNotification.jsx:21]: These should be constants.

**Comment by alisaad673** [comment]
Needs conflict resolution

**Comment by mavisakalyan** [comment]
Got it will get right on it.

**Comment by avaer** [comment]
This pr does not follow point 3 on the checklist.

**Pull request merged**